src/qt/coincontroldialog.cpp

```diff
@@ -607,6 +596,31 @@ void CoinControlDialog::updateLabels()
         // after fee
         nAfterFee = std::max<CAmount>(nAmount - nPayFee, 0);
     }
+}
+
+void CoinControlDialog::updateLabels()
+{
+    if (!model)
+        return;
+
+    ui->labelTitle->setText(fSelectTransparent ?
+            "Select PIV Outputs to Spend" :
+            "Select Shielded PIV to Spend");
+
+    CAmount nPayAmount = 0;
+    CAmount nAmount = 0;
+    CAmount nPayFee = 0;
+    CAmount nAfterFee = 0;
+    CAmount nChange = 0;
+    unsigned int nQuantity = 0;
+    unsigned int nBytes = 0;
+    bool fDust = false;
+    getTotals(nPayAmount, nAmount, nPayFee, nAfterFee, nChange, nQuantity, nBytes, fDust);
```

Comment:
future PR work: wouldn't hurt to return an struct in `getTotals`, the values are cleared inside the function anyway.